Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -130,10 +130,43 @@
         "maxSharedHits",
         "maxChi2",
         "nMeasurementsGroupMin",
+        "requireReferenceSurface",
     ],
-    defaults=[(None, None)] * 7 + [None] * 7,
+    defaults=[(None, None)] * 7 + [None] * 8,
 )
 
+
+# Encapsulate this boilerplate code into a function so different uses do not get out of sync
+# Gets an additional lambda for the etaMax, because we might want to use this with eta binning
```

Comment:
I believe the doc needs to be in the function?

Reply:
true, don't know why I did it this way^^